api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -521,8 +526,128 @@ public void removePatientFromCohort_shouldSaveCohortAfterRemovingPatient() throw
 		// make a patient, add it using the method
 		Patient patientToAddThenRemove = new Patient(4);
 		service.addPatientToCohort(service.getCohort(2), patientToAddThenRemove);
-		assertTrue(service.getCohort(2).contains(patientToAddThenRemove));
+		assertTrue(service.getCohort(2).contains(patientToAddThenRemove.getPatientId()));
 		service.removePatientFromCohort(service.getCohort(2), patientToAddThenRemove);
-		assertFalse(service.getCohort(2).contains(patientToAddThenRemove));
+		List<CohortMembership> memberList = service.getCohort(2)
+				.getMembers().stream()
+				.filter(m -> m.getPatient().getPatientId().equals(patientToAddThenRemove.getPatientId()))
+				.collect(Collectors.toList());
+		CohortMembership memberWithPatientToRemove = memberList.get(0);
+		assertNotNull(memberWithPatientToRemove.getEndDate());
+	}
+
+	/**
+	 * @verifies {@link CohortService#addMembershipToCohort(Cohort, CohortMembership)}
+	 * test = should add membership to cohort
```

Comment:
I do not see any value added by it. It just gives us more unneeded lines to maintain. So just get rid of it. :)